### ISSUE TITLE
EMA C++: NIProvider doesn't encode the LoadFilter to the directory refresh message

### ISSUE THREAD

**Issue by jirapongse-refinitiv** [Refinitiv/Real-Time-SDK#287]
I ran the NIProv100 example and specified the LoadFilter in the configuration file.

```
				</StateFilter>
				<LoadFilter>
					<OpenLimit value="100"/>					
				</LoadFilter>
			</Service>

			
		</Directory>

```

EMA C++ (RTSDK 2.2.1.L1) doesn't encode the LoadFilter to the Directory refresh message.

**Comment by fogol** [comment]
@jirapongse-refinitiv Thank you for bringing this to our attention. We will investigate the issue to provide the solution.

**Comment by vlevendel** [comment]
@jirapongse-refinitiv Fixed with version Real-Time-SDK-2.2.3.L1. Please reopen if there are additional concerns.
